reporter/src/main/resources/templates/notice/default.ftl

```diff
@@ -67,14 +67,18 @@ ${exceptionText}
     [/#if]
 This software depends on external packages and source code.
 The applicable license information is listed below:
-[/#if]
 
+[/#if]
```

Comment:
is this change intentional?

Reply:
Yes, IIRC this new blank line inside the `[/#if]` is a substitute for the blank line in former line 69 outside the `[/#if]`. TBH, I also only found out by running the tests and tweaking the output.